src/main/java/calculator/calculator/StringCalculator.java

```diff
@@ -0,0 +1,39 @@
+package calculator.calculator;
+
+import calculator.global.ErrorMessage;
+import java.math.BigInteger;
+import java.util.List;
+import java.util.Objects;
+
+public class StringCalculator implements Calculator {
+
+    private final TokenProvider tokenProvider;
+
+    public StringCalculator(TokenProvider tokenProvider) {
+        this.tokenProvider = tokenProvider;
+    }
+
+    @Override
+    public BigInteger calculate(String rawInput) {
+        validateInput(rawInput);
+        List<String> tokens = tokenProvider.getTokens(rawInput);
+
+        if (isAllBlank(tokens)) {
+            return BigInteger.ZERO;
+        }
+
+        return tokens.stream()
+                .map(BigInteger::new)
+                .reduce(BigInteger.ZERO, BigInteger::add);
+    }
+
+    private void validateInput(String input) {
+        if (Objects.isNull(input)) {
```

Comment:
```suggestion
        if (input == null) {
```
이렇게 작성할 수도 있는데 `Objects.isNull()` 메서드를 사용한 이유가 무엇인가요?

Reply:
가독성 면에서 좀 더 __문장처럼__ 읽힌다는 장점이 있다고 생각했습니다!